src/bundle/Resources/views/content/tab/url/modal_add_customurl.html.twig

```diff
@@ -0,0 +1,42 @@
+{% trans_default_domain 'locationview' %}
+{% form_theme form 'EzPlatformAdminUiBundle:content:form_fields.html.twig' '@EzPlatformAdminUi/content/tab/url/checkbox.html.twig' %}
+
+<div class="modal fade ez-modal ez-modal--custom-url-alias" id="ez-modal--custom-url-alias" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+        {{ form_start(form, {'action': path('ezplatform.customurl.add')}) }}
+        <div class="modal-content">
+            <div class="modal-header">
+                <h3 class="modal-title">{{ 'tab.urls.add.title'|trans|desc('Create new URL alias') }}</h3>
+                <button type="button" class="close" data-dismiss="modal" aria-label="{{ 'translation.modal_close'|trans|desc('Close') }}">
+                    <svg class="ez-icon ez-icon--medium" aria-hidden="true">
+                        <use xlink:href="{{ asset('bundles/ezplatformadminui/img/ez-icons.svg') }}#discard"></use>
+                    </svg>
+                </button>
+            </div>
+            <div class="modal-body">
+                <span class="ez-modal--custom-url-alias__label required">{{ 'tab.urls.add.path'|trans|desc('URL') }}</span>
+                {{ form_widget(form.path, {'attr': {'class': 'required'}}) }}
```

Comment:
If I'm not mistaked `required` class should be added when the type has option `required` set to true? Can you check that?